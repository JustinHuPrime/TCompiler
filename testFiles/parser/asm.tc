module foo;

void bar() {
  asm "syscall\n\tub2";
}