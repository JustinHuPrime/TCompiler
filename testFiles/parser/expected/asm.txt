testFiles/parser/input/asm.tc (code):
FILE(1, 1, STAB(ENTRY(bar, FUNCTION(testFiles/parser/input/asm.tc, 3, 1, void()))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), FUNDEFN(3, 1, KEYWORDTYPE(3, 1, void), ID(3, 6, bar, REFERENCES(testFiles/parser/input/asm.tc, 3, 1)), STAB(), COMPOUNDSTMT(3, 12, STAB(), ASMSTMT(4, 3, syscall\n\tub2))))
